src/main/java/spireMapOverhaul/zones/ambushed/AmbushedZone.java

```diff
@@ -0,0 +1,384 @@
+                                new Cultist(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_JAW_SLIME, 1, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new JawWorm(-1000.0F, 0),
+                                new AcidSlime_M(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_THIEF_SLIME, 1, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new SpikeSlime_M(-1000.0F, 0),
+                                new Looter(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_JAW_LOUSE, 1, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new JawWorm(-1000.0F, 0),
+                                new LouseNormal(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_THIEVES, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Looter(-1000.0F, 0),
+                                new Mugger(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_AVOCADO_RAT, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new ShelledParasite(-1000.0F, 0),
+                                new FungiBeast(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_SENTRY_BASEBALL, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Sentry(-1000.0F, 0),
+                                new SphericGuardian(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_POWER_COUPLE, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Centurion(-1000.0F, 0),
+                                new Healer(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_BYRD_CHOSEN, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Byrd(-1000.0F, 0),
+                                new Chosen(32.0F, 0)
+                        })),
+                new ZoneEncounter(AMBUSHED_BY_CULTIST_CHOSEN, 2, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Cultist(-1000.0F, 0),
+                                new Chosen(32.0F, 0)
+                        }))
+        );
+    }
+
+    @Override
+    public List<ZoneEncounter> getEliteEncounters() {
+        return Arrays.asList(
+                new ZoneEncounter(AMBUSHED_BY_SENTRIES, 1, () -> new MonsterGroup(
+                        new AbstractMonster[] {
+                                new Sentry(-1000.0F, 0),
+                                new Lagavulin(true)
```

Comment:
Maybe use the combatmodifying zone to check for the laga encounters and nerf it a bit? This does seem a lot harder than the base game elites

Reply:
I don't like the idea of having inconsistency in the game, where loading into a specific encounter nerfs a specific enemy. I'd rather try to find another enemy to replace Sentry with. I don't think the Sentry itself is that bad of an addition, it's the Dazes it adds that will REALLY mess a player up, especially if Laga debuffs them and their next turn is just full of Dazes.